package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@hmcts/cmc-draft-store-middleware",
+  "version": "1.0.0",
+  "description": "Node.js middleware for draft store",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "prepublishOnly": "tsc -p tsconfig.publish.json",
```

Comment:
is `-p` the same as `--project`? 
as project is being used in lint

Reply:
Yes, `-p` is a shortcut for `--project`. Will use longer version to make it more readable.